lib/membrane_audio_mixer.ex

```diff
@@ -78,13 +88,21 @@ defmodule Membrane.AudioMixer do
 
   @impl true
   def handle_init(%__MODULE__{caps: caps} = options) do
-    state =
-      options
-      |> Map.from_struct()
-      |> Map.put(:pads, %{})
-      |> then(&if caps == nil, do: &1, else: initialize_mixer_state(caps, &1))
+    if options.native_mixer && !options.prevent_clipping do
+      Membrane.Logger.error(
+        "Invalid element options, for native mixer only clipping preventing one is available"
+      )
 
-    {:ok, state}
+      {:error, :invalid_options}
+    else
+      state =
+        options
+        |> Map.from_struct()
+        |> Map.put(:pads, %{})
+        |> then(&if caps == nil, do: &1, else: initialize_mixer_state(caps, &1))
+
+      {:ok, state}
```

Comment:
Make sure mixer state is dropped when going to stopped to release native resources